src/applications/healthcare/questionnaire/selectors.js

```diff
@@ -0,0 +1,8 @@
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
+
+export const selectShowQuestionnaire = state =>
+  toggleValues(state)[FEATURE_FLAG_NAMES.showHealthcareExperienceQuestionnaire];
+
+export const selectLoadingFeatureFlags = state =>
+  state?.featureToggles?.loading;
```

Comment:
Nice! I didn't know this was a thing

Reply:
The Elvis operator makes null checks so much easier